kolibri/plugins/coach/assets/src/views/recent-page/channel-list.vue

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
For top-level components, we definitely want to use directory + index.vue.

For internal components, using a named vue file can be cleaner because the extra nesting is redundant.